proposals/instructor-training-selection.md

```diff
@@ -0,0 +1,169 @@
+# Instructor Training Selection
+
+**Warning: This is only a draft.**
+
+**Warning: This only apply for the two-day training course in December 2015.**
+
+## Terminology
+
+-   An **instructor** is someone who teaches a workshop. Instructor training (IT)
+    prepares them to do this.
+
+## Why Select Attends for Instructor Training
+
+SCF wants to train as more instructors as possible
+but demand for instructor training is higher than what SCF could offer
+at the moment.
+For the free/open online training SCF hopes to satisfy
+the demand for instructor training from their community
+that isn't associate with one partner or affiliate
+and could participate at the Instructor Training
+delivered for partners and affiliates.
```

Comment:
Suggest explicitly add "Therefore if you work for one of the SCF or DC Partnering Sites, please inquire first at your organisation if you could undergo IT hosted by them. The Instructor Training described in this document is addressed at people who do not have such affiliation and hence have pratcically no alternative for getting trained as an instructor."